runtime/compiler/env/J9ClassEnv.cpp

```diff
@@ -38,6 +38,8 @@
 #include "j9fieldsInfo.h"
 #include "rommeth.h"
 
+class TR_PersistentClassInfo;
```

Comment:
You need to include the header file with the definition of `TR_PersistentClassInfo`

Reply:
Okay, `TR_PersistentClassInfo` is defined in `RuntimeAssumptions.hpp`, added below line in this file. 
```
#include "runtime/RuntimeAssumptions.hpp"
```